### PR TITLE
Enhance pytest.ini settings.

### PR DESCRIPTION
**Pull request by berkerpeksag**
* Run only non-ES tests by default
* Pass -v by default
* Add some common directories to norecursedirs

### CODE REVIEW

File: pytest.ini [hunk 1]
```diff
@@ -1,5 +1,6 @@
 [pytest]
-addopts = --reuse-db --tb=native
+addopts = --reuse-db --tb=native -m 'not es_tests' -v
```
Comment (kmaglione) [inline pytest.ini]: The `-m 'not es_tests'` flag is used by `tox` to break the tests up into several parallel runs, to shorten the runtime of the test suite. It's not something we want as a default.

I also don't think `-v` is a good default. Users can add it when they think it makes sense. I don't think we should make that choice for them.

**Comment by kmaglione** [comment]
The `norecursedirs` change looks useful. r- on the others, though.

File: pytest.ini (hunk 1 above)
Comment (berkerpeksag) [inline pytest.ini]: Good point. Updated PR to only keep the norecursedirs line.

Thanks!

**Comment by magopian** [comment]
Looks useful, thanks @berkerpeksag !

r+

**Pull request merged**